src/app/content/experiments/selectors.ts

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+import * as parentSelectors from '../selectors';
+
+export const localState = createSelector(
+  parentSelectors.localState,
+  (parentState) => parentState.experiments
+);
+
+// how to create selectors for particular variants
+export const experimentsEnabled = createSelector(
+  localState,
+  (state) => state
+);
```

Comment:
@TomWoodward on Monday I'd like to talk about the best way to employ selectors to get the feature variant info we need

Reply:
yea so by design selectors can't take arguments, what you'd want is a selector that just returns all the flags and their values, and then maybe you could have a helper like `getVariantForFeatureFlag` or something that takes the flag name and internally uses the selector